src/DocumentsRepository/Traits/LDPDocumentsRepositoryTrait.spec.ts

```diff
@@ -2484,6 +2484,46 @@ describe( "LDPDocumentsRepositoryTrait", () => {
 				expect( returned ).toBe( document );
 			} );
 
+			it( "should not send request if no dirty", async () => {
+				stubRequest( "https://example.com/" );
+
+				await repository.save( document );
+
+				const requests:number = jasmine.Ajax.requests.count();
+				expect( requests ).toBe( 0, "A request was made when it should not" );
+			} );
+
+			it( "should not send request if no dirty with undefined", async () => {
+				stubRequest( "https://example.com/" );
+
+				Object.assign( document, { property: undefined } );
+				await repository.save( document );
+
+				const requests:number = jasmine.Ajax.requests.count();
+				expect( requests ).toBe( 0, "A request was made when it should not" );
+			} );
+
+			it( "should not send request if no dirty with null", async () => {
+				stubRequest( "https://example.com/" );
+
+				Object.assign( document, { property: undefined } );
```

Comment:
Test title says the document property got modified to `null` but test is changing it to `undefined`